airflow/www/views.py

```diff
@@ -313,25 +326,33 @@ def get_int_arg(value, default=0):
                     filename=filename),
                 "error")
 
-        num_of_all_dags = dags_query.count()
+        num_of_all_dags = current_dags.count()
         num_of_pages = int(math.ceil(num_of_all_dags / float(dags_per_page)))
 
+        status_count_all = all_dags.count()
+        status_count_active = active_dags.count()
+        status_count_paused = paused_dags.count()
```

Comment:
This does three queries - we could do it with two very easily as `count_all == count_active+count_paused`, right?

(We could _attempt_ to do it as one query with `SELECT is_paused, count(*) ... GROUP BY is_paused` but that may or may not be more efficient in the end)

Reply:
Updated (with your sum suggestion).